R/calculate_silhouette_width.R

```diff
@@ -3,7 +3,8 @@
 #' This function performs replicated calculations on downsampled data.
 #'
 #' @param merged_sce The merged SCE object containing data from multiple batches
-#' @param pc_name The name that allows access to the PCs. Example: fastMNN_PCA
+#' @param pc_list A list of names that allow access to the PCs in the merged SCE
+#'  object. Example: c("PCA", "fastMNN_PCA").
```

Comment:
This is not a list, but a vector! The argument name should better reflect the actual type. That said, `pc_vector` is not a very appealing argument name, so perhaps `pc_names`? This will require updating code in several places, and re-running document.

Reply:
I went with `pc_names`, updated the docs, and on my way to propagate this change to #199!